soa/service/remote_credential_provider.cc

```diff
@@ -1,188 +0,0 @@
-// This file is part of MLDB. Copyright 2015 Datacratic. All rights reserved.
```

Comment:
My feeling is that this is the type of complexity we should remove for now, even if we end up adding it back in later. I think we have many hurdles to clear before "lack of external creds provider" is an objection that stops any uptake.